.travis.yml

```diff
@@ -32,10 +32,10 @@ matrix:
       compiler: clang
 
     # iPhoneOS 64bit
-    #- env: TARGET=aarch64-apple-ios
-    #  os: osx
-    #  osx_image: xcode9.4
-    #  rust: nightly
+    - env: TARGET=aarch64-apple-ios
+      os: osx
+      osx_image: xcode11
```

Comment:
nit: it might be useful to keep the Xcode version low for backwards compatibility

Reply:
@grovesNL To clarify you mean adding both or **just** testing older ones?